### DISCUSSION TITLE
SuperUser Mission GUI Menu Items

### DISCUSSION THREAD

**Discussion by DragurNaut** [Security-Onion-Solutions/securityonion#10154] [2.4]
I just got SO up and running and my only user has superuser role. I wanted to edit the syslog filter to create different indexes for each device sending logs. My googling tells me there should be a configuration tab in the menu, which I do not have. I also only have "Users" under the administration tab and that same googling tells me I should have more options. Did I not install properly or am I missing something else?

**Comment by TOoSmOotH** [comment]
Sounds like you are running 2.3. Check your version in the grid screen or by running cat /etc/soversion on the manager

**Reply by DragurNaut** [reply]
Correct I am running Version: 2.3.230

Does this version need something extra?

**Reply by dougburks** [reply]
@notrednamc Version 2.4 has new features that were not available in 2.3.
